### PR TITLE
Out of Memory FAQ

### PR DESCRIPTION
**Pull request by beckywilks**
Hi, 

I added an FAQ for Out of Memory error messages. See #2038 

This is my first attempt at documentation so all advice is welcome. 

Becky :)

### CODE REVIEW

**Comment by jkwmoore** [comment]
As a side note, we can link PRs to issues with the development side tab on the PR page in Github:

image

**Comment by jkwmoore** [comment]
Apologies, I forgot to make my review fully live before you cracked on with amending. Making these visible now.

**Review by jkwmoore — CHANGES_REQUESTED** [review]

File: FAQs.rst [hunk 1]
```diff
@@ -710,4 +710,18 @@ but cannot use more than 400 at once.
 
 |br|
 
+"Out of Memory", "OOM" errors and job prematurely stopping
+----------------------------------------------------------
+
+For both interactive and batch jobs, resources are requested for the job. When 'Out of Memory' and 'OOM' errors occur it means not enough memory has been allocated for the job to run to completion.
```
Comment (nikishe) [inline FAQs.rst:716]:
```suggestion
When "Out of Memory" (OOM) errors occur in an interactive or batch  session , it indicates insufficient memory has been allocated for the job to run to completion.
```

**Review by nikishe — COMMENTED** [review]
Congratulation on your first pull request. I have added a small suggestion to make one of your sentences short and snappy

**Review by nikishe — APPROVED** [review]
LGTM

File: FAQs.rst [hunk 2]
```diff
@@ -710,4 +710,18 @@ but cannot use more than 400 at once.
 
 |br|
 
+"Out of Memory", "OOM" errors and job prematurely stopping
+----------------------------------------------------------
+
+When "Out of Memory" (OOM) errors occur in an interactive or batch  session , it indicates insufficient memory has been allocated for the job to run to completion.
```
Comment (Carldkennedy) [inline FAQs.rst:716]:
```suggestion
When "Out of Memory" (OOM) errors occur in an interactive or batch session, it indicates insufficient memory has been allocated for the job to run to completion.
```

**Review by Carldkennedy — APPROVED** [review]
LGTM

File: FAQs.rst [hunk 3]
```diff
@@ -728,4 +728,18 @@ but cannot use more than 400 at once.
 
 |br|
```
Comment (jkwmoore) [inline FAQs.rst:729]:
```suggestion
-----
```

Looks like we don't really need the additional break line but we are missing a separator.

**Review by jkwmoore — CHANGES_REQUESTED** [review]
Please merge when the separator is in place. Ta!

**Review by jkwmoore — APPROVED** [review]

**Review by nikishe — APPROVED** [review]

**Pull request closed without merging**